core_v2/sources/domain_e2e_tests.move

```diff
@@ -0,0 +1,890 @@
+#[test_only]
+module aptos_names_v2::domain_e2e_tests {
+    use aptos_framework::chain_id;
+    use aptos_framework::timestamp;
+    use aptos_framework::object;
+    use aptos_names_v2::config;
+    use aptos_names_v2::domains;
+    use aptos_names_v2::time_helper;
+    use aptos_names_v2::test_helper;
+    use aptos_names_v2::test_utils;
+    use std::option;
+    use std::signer;
+    use std::string;
+    use std::vector;
+
+    const MAX_REMAINING_TIME_FOR_RENEWAL_SEC: u64 = 15552000;
+
+    #[test(
```

Comment:
usually prefer we define the test execution at the top, that way when I read it, I challenge myself to equate the two...